### PR TITLE
Update docs to cover channel selection

### PR DESCRIPTION
**Pull request by fleetwoods**

### CODE REVIEW

File: src/main/scala/DoNotCommit.scala [hunk 1]
```diff
@@ -0,0 +1,83 @@
+class DoNotCommit {
```
Comment (laurence-bird) [inline src/main/scala/DoNotCommit.scala]: Guessing this wasn't meant to be committed?

File: src/main/tut/docs/channels.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+## Customer Communication Preferences
+
+Within My OVO customers set which channels they wish to receive comms over, they are able to do this for Service and Marketing comms separately.
+ 
+For Service comms the customer is forced to choose either Post or Email, with SMS available as an additional option.
+
+Unless specified by the customer in their communication preferences, comms will not be issued via said channel.
+
+<div class="alert alert-info">
+If a customer has no Service Communication Preferences this indicates that they have not actually set any, as either email or post must be selected, and therefore the customer's Comminication Preferences are not taked into account.
```
Comment (laurence-bird) [inline src/main/tut/docs/channels.md]: *not taken into account

Comment (cb372) [inline src/main/tut/docs/channels.md]: s/taked/taken

File: src/main/tut/docs/channels.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+</div>
+
+  
+## Customer Profile
+  
+The customer-service provides us with contact details for a customer.
```
Comment (cb372) [inline src/main/tut/docs/channels.md]: profiles service?

Comment (fleetwoods) [inline src/main/tut/docs/channels.md]: Name checked the actual service we pull the data from, but yeah when we own the data can update.

**Pull request merged**